tools/topology/topology2/include/pipelines/cavs/google-rtc-aec-capture.conf

```diff
@@ -89,26 +89,6 @@ Class.Pipeline."google-rtc-aec-capture" {
 			]
                 }
 
-		google-rtc-aec."1" {
-			Object.Control.bytes."1" {
```

Comment:
Does the real AEC version without mockup use the bytes control. Was this added just for the mockup? If the real AEC has bytes control shouldn't we have it in topology to be able to exercise it with about right ballpark size random bytes blob?